src/medCore/gui/toolboxes/medRegistrationSelectorToolBox.cpp

```diff
@@ -207,14 +178,13 @@ void medRegistrationSelectorToolBox::changeCurrentToolBox(int index)
     }
 }
 
-
 //! Clears the toolbox.
 void medRegistrationSelectorToolBox::clear(void)
```

Comment:
There is still one minor issue here:
```medSelectorToolBox``` clears the input, but this method doesn't call the base one, so that functionality is lost.